core/blockly.js

```diff
@@ -247,8 +263,11 @@ Blockly.onKeyDown_ = function(e) {
     } else if (e.keyCode == 90) {
       // 'z' for undo 'Z' is for redo.
       Blockly.hideChaff();
-      mainWorkspace.undo(e.shiftKey);
+      workspace.undo(e.shiftKey);
```

Comment:
@alschmiedt What's the reason for this line of change? It seems to have broken undo with Ctrl + Z.

Reply:
I think this just got incorrectly copied over from the rebase. I'll make a PR to fix now.